contribs/gmf/src/editing/editFeatureComponent.js

```diff
@@ -128,7 +130,8 @@ exports.component_ = function() {
       'map': '<gmfEditfeatureMap',
       'state': '=gmfEditfeatureState',
       'tolerance': '<?gmfEditfeatureTolerance',
-      'vectorLayer': '<gmfEditfeatureVector'
+      'vectorLayer': '<gmfEditfeatureVector',
+      'closeaftersave': '=?gmfEditfeatureCloseaftersave'
```

Comment:
To make it consistent you hould use camelcase: `closeAfterSave`.

Reply:
done